ovn-tester/ovn_workload.py

```diff
@@ -99,6 +101,10 @@ def enable_trim_on_compaction(self):
             self.phys_node.run(f'docker exec {db_container} ovs-appctl -t '
                                f'/run/ovn/ovnsb_db.ctl '
                                f'ovsdb-server/memory-trim-on-compaction on')
+        for relay_container in self.relay_containers:
+            self.phys_node.run(f'docker exec {db_container} ovs-appctl -t '
```

Comment:
This should be `relay_container` instead of `db_container`.

Reply:
Whoops, this is fixed now.